docs/plugins/rulesets.md

```diff
@@ -0,0 +1,59 @@
+# Repository Rulesets
+
+> [!WARNING]
+> Support for Repository Rulesets is still under development.
+> Details may still change, like how the configuration is defined in the `settings.yml`.
+> Please follow [#732](https://github.com/repository-settings/app/issues/732) for progress updates.
+> Feedback is appreciated, but adopt cautiously, with the expectation of breaking changes until support is fully released.
+
```

Comment:
while this should be fully functional with the current schema, it mirrors the github API expectations, which are somewhat cryptic for a non-programatic context. we prefer to mirror the github API most of the time, but i'm leaning away from that in this case

because of that, i want to release this progress in a semi-dark pre-release fashion. this warning is included in the docs to attempt to make that visible